# Norwegian Bokmål translation for apport
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-12-07 16:23+0100\n"
"PO-Revision-Date: 2015-04-27 21:47+0000\n"
"Last-Translator: Martin Pitt <martin@piware.de>\n"
"Language-Team: Norwegian Bokmål <nb@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-12-07 14:36+0000\n"
"X-Generator: Launchpad (build 41e7553f8097ce3a683f90e835708cbe7bf0006c)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Samle inn informasjon om systemet"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"Du må autentisere deg for å samle inn systeminformasjon til denne "
"feilrapporten"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Feilrapporter for system"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr "Skriv inn passordet ditt for å få tilgang til systemets feilrapporter"

#: ../apport/ui.py:165
msgid "This package does not seem to be installed correctly"
msgstr "Denne pakken ser ikke ut til å være korrekt installert"

#: ../apport/ui.py:175
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""

#: ../apport/ui.py:205
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Du har gamle pakkeversjoner installert. Oppgrader følgende pakker, og se om "
"feilen fremdeles oppstår:\n"
"\n"
"%s"

#: ../apport/ui.py:359
msgid "unknown program"
msgstr "ukjent program"

#: ../apport/ui.py:362
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Beklager, programmet \"%s\" ble uventet avsluttet"

#: ../apport/ui.py:365 ../apport/ui.py:1899
#, python-format
msgid "Problem in %s"
msgstr "Problem i %s"

#: ../apport/ui.py:370
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Datamaskinen din har ikke nok tilgjengelig minne for å kunne analysere og "
"rapportere problemet automatisk."

#: ../apport/ui.py:430 ../apport/ui.py:448 ../apport/ui.py:621
#: ../apport/ui.py:630 ../apport/ui.py:867 ../apport/ui.py:1677
#: ../apport/ui.py:1860 ../apport/ui.py:1865
msgid "Invalid problem report"
msgstr "Ugyldig problemrapport"

#: ../apport/ui.py:431
msgid "You are not allowed to access this problem report."
msgstr "Du har ikke tilgang til denne feilrapporten."

#: ../apport/ui.py:439
msgid "Error"
msgstr "Feil"

#: ../apport/ui.py:441
msgid "There is not enough disk space available to process this report."
msgstr "Det er ikke nok ledig diskplass til å behandle denne rapporten."

#: ../apport/ui.py:479
msgid "No PID specified"
msgstr ""

#: ../apport/ui.py:481
msgid "You need to specify a PID. See --help for more information."
msgstr ""

#: ../apport/ui.py:492 ../apport/ui.py:597
msgid "Invalid PID"
msgstr "Ugyldig prosess-id (PID)"

#: ../apport/ui.py:493
msgid "The specified process ID does not exist."
msgstr ""

#: ../apport/ui.py:498
msgid "Not your PID"
msgstr ""

#: ../apport/ui.py:499
msgid "The specified process ID does not belong to you."
msgstr ""

#: ../apport/ui.py:554
msgid "No package specified"
msgstr "Ingen pakke er spesifisert"

#: ../apport/ui.py:556
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr "Du må spesifisere en pakke eller PID: Se --hjelp for mer informasjon."

#: ../apport/ui.py:584
msgid "Permission denied"
msgstr "Adgang nektet"

#: ../apport/ui.py:586
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Den valgte prosessen tilhører ikke deg. Kjør dette programmet som eieren av "
"prosessen eller som root."

#: ../apport/ui.py:599
msgid "The specified process ID does not belong to a program."
msgstr "Den valgte prosess-IDen tilhører ikke noe program."

#: ../apport/ui.py:623
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Symptomskript %s fant ingen berørte pakker"

#: ../apport/ui.py:631
#, python-format
msgid "Package %s does not exist"
msgstr "Pakken %s finnes ikke"

#: ../apport/ui.py:660 ../apport/ui.py:872 ../apport/ui.py:905
#: ../apport/ui.py:915
msgid "Cannot create report"
msgstr "Kan ikke lage rapport"

#: ../apport/ui.py:675 ../apport/ui.py:734 ../apport/ui.py:752
msgid "Updating problem report"
msgstr "Oppdaterer problemrapport"

#: ../apport/ui.py:677
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Du er ikke en rapportør eller abonnent på denne problemrapporten, eller "
"rapporten er et duplikat eller allerede lukket.\n"
"\n"
"Vennligst lag en ny rapport ved å bruke «apport-bug»."

#: ../apport/ui.py:689
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Du er ikke opprinnelig innmelder av denne problemrapporten. Det er enklere å "
"merke en feil som et duplikat av en annen enn å flytte dine kommentarer og "
"vedlegg til en ny feil.\n"
"\n"
"Vi vil også anbefale deg å rapportere en ny feil ved å bruke «apport-bug» og "
"lage en kommentar i denne feilen om den du rapporterer.\n"
"\n"
"Ønsker du å fortsette?"

#: ../apport/ui.py:735 ../apport/ui.py:753
msgid "No additional information collected."
msgstr "Ingen tilleggsinformasjon har blitt samlet inn."

#: ../apport/ui.py:816
msgid "What kind of problem do you want to report?"
msgstr "Hva slags problem vil du rapportere?"

#: ../apport/ui.py:837
msgid "Unknown symptom"
msgstr "Ukjent symptom"

#: ../apport/ui.py:838
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Symptomet \"%s\" er ukjent."

#: ../apport/ui.py:874
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:890
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"Vennligst klikk på et programvindu etter at du har lukket denne meldingen, "
"slik at du kan sende en feilrapport om problemet."

#: ../apport/ui.py:907 ../apport/ui.py:916
msgid "xprop failed to determine process ID of the window"
msgstr "«xprop» fant ikke vinduets prosess-ID"

#: ../apport/ui.py:931
#, python-format
msgid "%(prog)s <report number>"
msgstr ""

#: ../apport/ui.py:932
msgid "Specify package name."
msgstr "Angi pakkenavn."

#: ../apport/ui.py:938 ../apport/ui.py:1070
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "Legg til et ekstra stikkord i rapporten. Kan nevnes flere ganger."

#: ../apport/ui.py:976
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""

#: ../apport/ui.py:987
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Start i feilrapporteringsmodus. Krever --package og en valgfri --pid, eller "
"kun --pid. Hvis ingen av disse er gitt vises en liste over kjente symptomer. "
"(Underforstått at ett argument blir gitt)"

#: ../apport/ui.py:996
msgid "Click a window as a target for filing a problem report."
msgstr "Klikk på det aktuelle vinduet for å sende en feilrapport om problemet."

#: ../apport/ui.py:1005
msgid "Start in bug updating mode. Can take an optional --package."
msgstr "Start i feiloppdateringmodus. Aksepterer en alternativ --pakke."

#: ../apport/ui.py:1014
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Send en feilmelding om et problem. (Underforstått hvis problemnavn er gitt "
"som eneste argument)"

#: ../apport/ui.py:1023
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Oppgi pakkenavn i --file-bug-modus. Dette er valgfritt hvis --pid er gitt. "
"(Underforstått hvis pakkenavn er gitt som eneste argument.)"

#: ../apport/ui.py:1034
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Spesifiser et kjørende program i «--file-bug»-modus. Hvis dette er "
"spesifisert, vil feilrapporten inneholde mer informasjon. (Dette er allerede "
"implisert hvis «pid» oppgis som eneste argument.)"

#: ../apport/ui.py:1042
msgid "The provided pid is a hanging application."
msgstr "Den oppgitte pid-en er en applikasjon som henger."

#: ../apport/ui.py:1050
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Rapporter kræsj fra en gitt .apport- eller .crash-fil i stedet for de "
"gjeldende i %s. (Underforstått hvis en fil er gitt som eneste argument)."

#: ../apport/ui.py:1060
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"I feilrapporteringsmodus kan du lagre den samlede feilinformasjonen i en fil "
"i stedet for å rapportere den. Filen kan innrapporteres senere, for eksempel "
"fra en annen maskin."

#: ../apport/ui.py:1078
msgid "Print the Apport version number."
msgstr "Skriv ut Apport-versjonsnummeret."

#: ../apport/ui.py:1240
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""
"Dette vil starte apport-retrace i et terminalvindu for å undersøke krasjen."

#: ../apport/ui.py:1244
msgid "Run gdb session"
msgstr "Kjør gdb-økt"

#: ../apport/ui.py:1245
msgid "Run gdb session without downloading debug symbols"
msgstr "Kjør gdb-økt uten å laste ned feilrapporteringssymboler"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1247
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Oppdater %s med symbolsk stabelsporing"

#: ../apport/ui.py:1309
msgid "Can't remember send report status settings"
msgstr ""

#: ../apport/ui.py:1313
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""

#: ../apport/ui.py:1396 ../apport/ui.py:1409
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""
"Denne problemrapporten gjelder et program som ikke lenger er installert."

#: ../apport/ui.py:1433
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"Problemet skjedde med programmet %s, som har endret seg siden kræsjen "
"oppstod."

#: ../apport/ui.py:1494 ../apport/ui.py:1618 ../apport/ui.py:1869
msgid "This problem report is damaged and cannot be processed."
msgstr "Denne feilrapporten er skadet, og kan ikke behandles."

#: ../apport/ui.py:1503
msgid "This report is about a package that is not installed."
msgstr "Denne rapporten gjelder en pakke som ikke er installert."

#: ../apport/ui.py:1511
msgid "An error occurred while attempting to process this problem report:"
msgstr "En feil oppstod da denne feilrapporten skulle behandles:"

#: ../apport/ui.py:1528
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""

#: ../apport/ui.py:1533
#, python-format
msgid "%s snap"
msgstr ""

#: ../apport/ui.py:1534
#, python-format
msgid "%s deb package"
msgstr ""

#: ../apport/ui.py:1575
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""

#: ../apport/ui.py:1580
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""

#: ../apport/ui.py:1679
msgid "Could not determine the package or source package name."
msgstr "Kunne ikke bestemme pakken eller kildepakkens navn."

#: ../apport/ui.py:1704
msgid "Unable to start web browser"
msgstr "Kunne ikke starte nettleser"

#: ../apport/ui.py:1705
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Kunne ikke starte nettleseren for å åpne %s."

#: ../apport/ui.py:1815
msgid "Network problem"
msgstr "Nettverksfeil"

#: ../apport/ui.py:1819
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Kan ikke koble til krasjdatabasen, vennligst sjekk internettforbindelsen."

#: ../apport/ui.py:1851
msgid "Memory exhaustion"
msgstr "Minne fullt"

#: ../apport/ui.py:1853
msgid "Your system does not have enough memory to process this crash report."
msgstr "Systemet ditt har ikke nok minne til å behandle denne krasjrapporten."

#: ../apport/ui.py:1904
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Problemet kan ikke rapporteres:\n"
"\n"
"%s"

#: ../apport/ui.py:1960 ../apport/ui.py:1972
msgid "Problem already known"
msgstr "Problemet er allerede kjent"

#: ../apport/ui.py:1962
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Dette problemet er allerede rapportert i feilrapporten som vises i "
"nettleseren. Hvis du har mer informasjon som kan hjelpe utviklerne å løse "
"problemet, bør du legge det til i rapporten."

#: ../apport/ui.py:1974
msgid "This problem was already reported to developers. Thank you!"
msgstr "Dette problemet er allerede rapportert til utviklerne. Takk!"

#: ../bin/apport-cli.py:84
msgid "Press any key to continue..."
msgstr "Trykk en tast for å fortsette..."

#: ../bin/apport-cli.py:91
msgid "What would you like to do? Your options are:"
msgstr "Hva vil du gjøre? Alternativene dine er:"

#: ../bin/apport-cli.py:104
#, python-format
msgid "Please choose (%s):"
msgstr "Vennligst velg (%s):"

#: ../bin/apport-cli.py:172
#, python-format
msgid "(%i bytes)"
msgstr "(%i bytes)"

#: ../bin/apport-cli.py:174 ../gtk/apport-gtk.py:169 ../kde/apport-kde.py:406
msgid "(binary data)"
msgstr "(binære data)"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.py:208 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:199
msgid "Send problem report to the developers?"
msgstr "Send problemrapport til utviklerne?"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Etter at problemrapporten har blitt sendt: vennligst fyll ut skjemaet i den\n"
"automatisk åpnede nettleseren."

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "&Send rapport (%s)"

#: ../bin/apport-cli.py:223
msgid "&Examine locally"
msgstr "&Undersøk lokalt"

#: ../bin/apport-cli.py:227
msgid "&View report"
msgstr "&Vis rapport"

#: ../bin/apport-cli.py:230
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"&Behold rapportfilen for å sende den senere eller kopiere den til et annet "
"sted"

#: ../bin/apport-cli.py:235
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Avbryt og &ignorer framtidige krasj i denne programversjonen"

#: ../bin/apport-cli.py:238 ../bin/apport-cli.py:332 ../bin/apport-cli.py:366
#: ../bin/apport-cli.py:389
msgid "&Cancel"
msgstr "&Avbryt"

#: ../bin/apport-cli.py:273
msgid "Problem report file:"
msgstr "Problemrapportfil:"

#: ../bin/apport-cli.py:279 ../bin/apport-cli.py:284
msgid "&Confirm"
msgstr "&Bekreft"

#: ../bin/apport-cli.py:283
#, python-format
msgid "Error: %s"
msgstr "Feil: %s"

#: ../bin/apport-cli.py:289 ../kde/apport-kde.py:462
msgid "Collecting problem information"
msgstr "Samler probleminformasjon"

#: ../bin/apport-cli.py:291
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Den innsamlede informasjonen kan sendes til utviklerne for å forbedre\n"
"programmet. Dette kan ta et par minutter."

#: ../bin/apport-cli.py:307 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:511
msgid "Uploading problem information"
msgstr "Laster opp probleminformasjon"

#: ../bin/apport-cli.py:309
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Den innsamlede informasjonen sendes til feilsporingssystemet.\n"
"Dette kan ta et par minutter."

#: ../bin/apport-cli.py:365
msgid "&Done"
msgstr "&Fullført"

#: ../bin/apport-cli.py:371
msgid "none"
msgstr "ingen"

#: ../bin/apport-cli.py:373
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Valgt: %s. Flere valg:"

#: ../bin/apport-cli.py:390
msgid "Choices:"
msgstr "Valg:"

#: ../bin/apport-cli.py:404
msgid "Path to file (Enter to cancel):"
msgstr "Filbane (Enter for å avbryte):"

#: ../bin/apport-cli.py:410
msgid "File does not exist."
msgstr "Filen eksisterer ikke."

#: ../bin/apport-cli.py:412
msgid "This is a directory."
msgstr "Dette er en mappe."

#: ../bin/apport-cli.py:418
msgid "To continue, you must visit the following URL:"
msgstr "For å fortsette, må du gå til følgende URL:"

#: ../bin/apport-cli.py:421
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Du kan starte en nettleser nå, eller kopiere denne URL-en til en nettleser "
"på en annen maskin."

#: ../bin/apport-cli.py:427
msgid "Launch a browser now"
msgstr "Start nettleseren nå"

#: ../bin/apport-cli.py:443
msgid "No pending crash reports. Try --help for more information."
msgstr "Ingen ventende krasjrapporter. Prøv --help for mer informasjon."

#: ../bin/apport-retrace.py:44
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "Legg ikke til flere meldinger i rapporten, skriv de heller til stdout."

#: ../bin/apport-retrace.py:53
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Start en interaktiv gdb-sesjon med rapportens kjernedump (-o ignoreres; "
"skriver ikke rapporten på nytt)"

#: ../bin/apport-retrace.py:62
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Skriv den endrede rapporten til en gitt fil i stedet for å andre den "
"opprinnelige rapporten"

#: ../bin/apport-retrace.py:72
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""
"Fjern kjerne-informasjon fra rapporten etter \"stack trace\" er opprettet på "
"nytt"

#: ../bin/apport-retrace.py:80
msgid "Override report's CoreFile"
msgstr "Overstyr rapportens kjernefil"

#: ../bin/apport-retrace.py:86
msgid "Override report's ExecutablePath"
msgstr "Overstyr rapportens kjøresti"

#: ../bin/apport-retrace.py:92
msgid "Override report's ProcMaps"
msgstr "Overstyr rapportens ProcMaps"

#: ../bin/apport-retrace.py:98
msgid "Rebuild report's Package information"
msgstr "Gjennoppbygg pakkeinformasjonen i rapporten"

#: ../bin/apport-retrace.py:105
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Bygg en midlertidig sandkasse, og last ned/installer nødvendige pakker og "
"feilsøkingssymboler i den. Uten dette valget, antas de nødvendige pakkene og "
"feilsøkingssymbolene å være installert i systemet allerede. Argumentet peker "
"til pakkesystemets grunnmappe. Hvis du spesifiserer «system», blir systemets "
"konfigurasjonsfiler brukt, men da vil du bare kunne spore en kræsj hvis den "
"oppstod på utgivelsen som kjører nå."

#: ../bin/apport-retrace.py:119
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""
"Sett opp en midlertidig sandkasse og installer samme versjon av gdb + "
"tilbehør som rapporten i stedet for versjonen av gdb du har installert nå."

#: ../bin/apport-retrace.py:129
msgid "Report download/install progress when installing packages into sandbox"
msgstr ""
"Rapporter nedlastings- og installasjonsfremdrift når pakker blir installert "
"i sandkassa."

#: ../bin/apport-retrace.py:136
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "Sett tidsstempler i starten av logg-meldinger, for bunthåndtering"

#: ../bin/apport-retrace.py:142
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr "Lag og bruk tredjepartslager fra kilder som er valgt i rapporter"

#: ../bin/apport-retrace.py:150
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "Legg pakkenedlastinger for sandkassa i hurtiglageret"

#: ../bin/apport-retrace.py:156
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Mappe for utpakkede pakker. Senere kjøringer vil basere seg på at nedlastede "
"pakker også har blitt pakket ut til denne sandkassa."

#: ../bin/apport-retrace.py:167 ../bin/apport-valgrind.py:97
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr "Installer en ekstrapakke i sandkassa (kan spesifiseres flere ganger)"

#: ../bin/apport-retrace.py:174
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Sti til en fil med autentifiseringsinformasjon for krasjdatabasen. Dette "
"brukes til å spesifisere en krasj-ID når krasjrapporten lastes opp (kun hvis "
"hverken -g, -o, eller -s spesifiseres)"

#: ../bin/apport-retrace.py:184
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Vis krasjrapporten og be om bekreftelse før den sendes til krasjdatabasen."

#: ../bin/apport-retrace.py:192
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr "Sti til kopien av sqlite-databasen. (Forvalgt: ikke se etter kopier)"

#: ../bin/apport-retrace.py:200
msgid "Do not add StacktraceSource to the report."
msgstr "Ikke ta med StacktraceSource i feilrapport."

#: ../bin/apport-retrace.py:213
msgid "You cannot use -C without -S. Stopping."
msgstr "Du kan ikke bruke -C uten -S. Stopper."

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:243
msgid "OK to send these as attachments? [y/n]"
msgstr "OK å sende disse som vedlegg? [j/n]"

#: ../bin/apport-unpack.py:29
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:31
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:33
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:59
msgid "Destination directory exists and is not empty."
msgstr "Målmappe finnes og er ikke tom."

#: ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "Se programmets bruksanvisning («man»-side) for mer informasjon."

#: ../bin/apport-valgrind.py:46
msgid "specify the log file name produced by valgrind"
msgstr "spesifiser navn på loggfilen som produseres av valgrind"

#: ../bin/apport-valgrind.py:52
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"bruk en tidligere opprettet sandkassemappe (SDIR), eller opprett den hvis "
"den ikke eksisterer"

#: ../bin/apport-valgrind.py:60
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"Bruk kun installerte feilsøkingssymboler, i stedet for å opprette eller "
"bruke en eksisterende sandkassemappe for flere symboler."

#: ../bin/apport-valgrind.py:69
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"bruk en tidligere opprettet hurtiglagermappe (CDIR), eller opprett en hvis "
"det ikke eksisterer"

#: ../bin/apport-valgrind.py:78
msgid "report download/install progress when installing packages into sandbox"
msgstr ""
"rapporter nedlastings- og installasjonsfremdrift når pakker installeres i "
"sandkassa"

#: ../bin/apport-valgrind.py:86
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:129
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Feil: %s er ikke et kjørbart program. Stopper."

#: ../data/apportcheckresume.py:68
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""
"Dette oppstod mens maskinen stod i hvilemodus, og hindret systemet fra å "
"våkne skikkelig."

#: ../data/apportcheckresume.py:73
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""
"Dette oppstod mens maskinen stod i dvalemodus, og hindret systemet fra å "
"våkne skikkelig."

#: ../data/apportcheckresume.py:81
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Gjenopptagelsen hang seg opp nær slutten og det vil synes som om denne "
"fullførte normalt."

#: ../data/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr "Systemet ditt kan bli ustabilt nå, og kan måtte startes på nytt."

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "Rapporter et problem"

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Rapporter en feil til utviklerne"

#: ../gtk/apport-gtk.py:185
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Beklager. Programmet %s har fått en uventet stans."

#: ../gtk/apport-gtk.py:188
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Beklager. %s har lukket seg uventet."

#: ../gtk/apport-gtk.py:193 ../kde/apport-kde.py:214 ../kde/apport-kde.py:266
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Beklager, det har skjedd en intern feil med %s."

#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:207
msgid "Send"
msgstr "Send"

#: ../gtk/apport-gtk.py:234 ../gtk/apport-gtk.py:666 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:330
msgid "Show Details"
msgstr "Vis detaljer"

#: ../gtk/apport-gtk.py:247 ../kde/apport-kde.py:258 ../kde/apport-kde.py:276
msgid "Continue"
msgstr "Fortsett"

#: ../gtk/apport-gtk.py:271
#, python-format
msgid "The application %s has stopped responding."
msgstr "Programmet %s svarer ikke."

#: ../gtk/apport-gtk.py:275
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "Programmet \"%s\" svarer ikke."

#: ../gtk/apport-gtk.py:292 ../kde/apport-kde.py:222
#, python-format
msgid "Package: %s"
msgstr "Pakke: %s"

#: ../gtk/apport-gtk.py:298 ../kde/apport-kde.py:229
msgid "Sorry, a problem occurred while installing software."
msgstr "Beklager. Et problem oppstod under installasjon av programvare."

#: ../gtk/apport-gtk.py:310 ../gtk/apport-gtk.py:334 ../kde/apport-kde.py:238
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "Programmet %s har hatt en intern feil."

#: ../gtk/apport-gtk.py:316 ../kde/apport-kde.py:245
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "Programmet %s lukket seg uventet."

#: ../gtk/apport-gtk.py:347 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:271
msgid "If you notice further problems, try restarting the computer."
msgstr "Prøv å starte datamaskinen på nytt hvis det oppstår flere problemer"

#: ../gtk/apport-gtk.py:352 ../kde/apport-kde.py:278
msgid "Ignore future problems of this type"
msgstr "Ignorer denne typen problemer i fremtiden"

#: ../gtk/apport-gtk.py:670 ../kde/apport-kde.py:327
msgid "Hide Details"
msgstr "Skjul detaljer"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:552 ../kde/apport-kde.py:595
msgid "Apport"
msgstr "Apport"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "Avbryt"

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "OK"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "Krasjrapport"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>Beklager, det oppsto en intern feil.</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr "Husk dette i framtiden"

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "Ignorer fremtidige problemer med denne programversjonen"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr "Start programmet på nytt"

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "_Undersøk lokalt"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr "Ikke send"

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Samler inn informasjon om problemet</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Informasjon som kan hjelpe utviklerne med å reparere problemet du "
"rapporterer samles inn."

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Laster opp feilinformasjon</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:513
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Den innsamlede informasjonen sendes til feilsporingssystemet. Dette kan ta "
"et par minutter."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport-krasjfil"

#: ../kde/apport-kde.py:261
msgid "Leave Closed"
msgstr "Forbli lukket"

#: ../kde/apport-kde.py:262 ../kde/apport-kde.py:432
msgid "Relaunch"
msgstr "Start på nytt"

#: ../kde/apport-kde.py:355
msgid "Username:"
msgstr "Brukernavn:"

#: ../kde/apport-kde.py:356
msgid "Password:"
msgstr "Passord:"

#: ../kde/apport-kde.py:461
msgid "Collecting Problem Information"
msgstr "Samler informasjon om feil"

#: ../kde/apport-kde.py:464
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"Den innsamlede informasjonen kan sendes til utviklerne for å forbedre "
"programmet. Dette kan ta noen minutter."

#: ../kde/apport-kde.py:510
msgid "Uploading Problem Information"
msgstr "Laster opp informasjon om feil"
